graylog2-web-interface/src/components/sidecars/configuration-forms/ImportsViewModal.jsx

```diff
@@ -0,0 +1,145 @@
+import { Alert, Button, Modal, OverlayTrigger, Tooltip } from 'react-bootstrap';
+
+import BootstrapModalWrapper from 'components/bootstrap/BootstrapModalWrapper';
+import CombinedProvider from 'injection/CombinedProvider';
+import { PaginatedList, Spinner, Timestamp } from 'components/common';
+import UserNotification from 'util/UserNotification';
+
+const { CollectorConfigurationsActions } = CombinedProvider.get('CollectorConfigurations');
+
+class ImportsViewModal extends React.Component {
+  static propTypes = {
+    onApply: PropTypes.func.isRequired,
+  };
+
+  static initialState = {
+    uploads: undefined,
+    totalUploads: 0,
+    pagination: {
+      page: 1,
+    },
+  };
+
+  constructor(props) {
+    super(props);
+    this.state = ImportsViewModal.initialState;
+  }
+
+  PAGE_SIZE = 10;
+
+  open = () => {
+    this._loadUploads(this.state.pagination.page);
+    this.uploadsModal.open();
+  };
+
+  hide = () => {
+    this.uploadsModal.close();
+  };
+
+  _isLoading = () => {
+    return !this.state.uploads;
+  };
+
+  _loadUploads = (page) => {
+    CollectorConfigurationsActions.listUploads({ page: page })
+      .then(
+        (uploads) => {
+          this.setState({ uploads: uploads.uploads, totalUploads: uploads.total });
+        },
+        (error) => {
+          this.setState({ uploads: [], totalUploads: 0 });
+          UserNotification.error(`Fetching uploads failed with error: ${error}`,
+            'Could not get configuration uploads');
+        },
+      );
+  };
+
+  _onApplyButton = (selectedUpload) => {
+    this.props.onApply(selectedUpload);
+  };
+
+  _buildVariableName = (name) => {
+    return `\${sidecar.${name}}`;
+  };
+
+  _formatUpload(upload) {
+    const tooltip = <Tooltip id={`${upload.id}-status-tooltip`}>{upload.collector_id}</Tooltip>;
+
+    return (
+      <tr key={upload.id}>
+        <td>
+          <OverlayTrigger placement="top" overlay={tooltip} rootClose>
+            <span>{upload.node_id}</span>
+          </OverlayTrigger>
+        </td>
+        <td>{upload.collector_name}</td>
+        <td><Timestamp dateTime={upload.created} format={'YYYY-MM-DD HH:mm:ss z'} /></td>
+        <td>
+          <Button bsStyle="info" bsSize="xsmall" onClick={() => this._onApplyButton(upload.rendered_configuration)}>
+            Apply
+          </Button>
+        </td>
+      </tr>
+    );
+  }
+
+  _formatModalBody() {
+    if (this._isLoading()) {
+      return (
+        <Spinner />
+      );
+    }
+
+    const pageSize = this.PAGE_SIZE;
+    const { uploads, totalUploads } = this.state;
+    const formattedUploads = uploads.map(upload => (this._formatUpload(upload)));
+
+    if (totalUploads === 0) {
+      return (
+        <Alert bsStyle="info">
+          <i className="fa fa-info-circle" />&nbsp;
+          There are no configuration uploads available. Please go to<br />
+          <strong>System-&gt;Collectors (legacy)-&gt;Details-&gt;Import Configuration</strong><br />
```

Comment:
This line is hard to read, as the whole line is bold and there are almost no spaces between words. We should add spaces between arrows and words, and maybe make it cursive instead.

Reply:
I think we should also remove those `br`s around this line. In the end we are already highlighting the way to get the configurations imported, so let's let the browser to break the line if it should.